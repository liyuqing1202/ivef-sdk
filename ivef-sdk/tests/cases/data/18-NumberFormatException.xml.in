<MSG_IVEF>
<Header Version="0.1.5" MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B"/>
<Body>
<ObjectDatas>
<ObjectData>
<TrackData Id="2" SourceId="12345" SourceName="abc" UpdateTime="2010-04-28T15:26:32.250Z" SOG="0,000000" COG="0,000000" Length="10,000000" TrackStatus="3">
<Pos Lat="0,000000" Long="0,000000" Altitude="10.0000"/>
</TrackData>
</ObjectData>
</ObjectDatas>
</Body>
</MSG_IVEF>
