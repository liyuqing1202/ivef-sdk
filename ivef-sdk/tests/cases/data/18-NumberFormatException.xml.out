TestApp ready for input
MSG_IVEF
    Header
        MsgRefId = 6289936C-DFAC-11E0-B7BA-B2E24824019B
        Version = 0.1.5
    Body
	  ObjectDatas
	      ObjectData
            	TrackData
                   COG = 0.0
                   Id = 2
                   Length = 10.000000
                   SOG = 0.0
		   SourceId = 12345
                   SourceName = abc
                   UpdateTime = 2010-04-28T15:26:32.250Z
		   TrackStatus = 3
                   Pos
		       Altitude=10.000000
                       Lat = 0.00000
                       Long = 0.00000
<MSG_IVEF>
<Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="0.1.5"/>
<Body>
<ObjectDatas>
<ObjectData>
<TrackData COG="0.0" Id="2" Length="10.000000" SOG="0.0" SourceId="12345" SourceName="abc" UpdateTime="2010-04-28T15:26:32.250Z" TrackStatus="3">
<Pos Altitude="10.000000" Lat="0.00000" Long="0.00000"/>
</TrackData>
</ObjectData>
</ObjectDatas>
</Body>
</MSG_IVEF>

TestApp shutting down
